pkg/epp/scheduling/framework/plugins/scorer/lora_affinity.go

```diff
@@ -27,6 +27,8 @@ import (
 
 const (
 	LoraAffinityScorerType = "lora-affinity-scorer"
+	ActiveModelsKey        = "ActiveModels"
+	WaitingModelsKey       = "WaitingModels"
```

Comment:
these are already defined (atm in plugins.go).
I think we should have metrics definitions in a centralized place.